Packs/PasswordResetViaChatbot/Playbooks/Reset_User_Password_via_Chatbot_README.md

```diff
@@ -0,0 +1,77 @@
+This playbook resets the password of an Active Directory or Okta user. The process is as follows:
+1. A user requests a password reset using a chatbot on Slack or on Microsoft Teams.
+2. The playbook optionally seeks approval to reset the user's password from the relevant stakeholder, or from the user's manager.
+3. If the reset was approved, the playbook optionally verifies the user using 2-factor authentication, from the user's available 2FA methods on Okta.
+4. If the reset is verified, a new password will be created while meeting the complexity requirements of the organization.
+5. The user's password will be reset and set to the newly generated password. The user will be forced to change their password on next login.
+6. The new user's password will be placed inside a password-protected ZIP (protected by a different password).
+7. The encrypted ZIP file that contains the new password for the user will be sent to the user via email. 
+8. The password for the ZIP file that contains the new user's password, will be sent to the requesting user through Slack or Teams.
+
+This playbook is intended for use with Slack or Teams. In order to use it, please make sure that you have a classifier and mapper in place. The classifier should create a Reset User Password via Chatbot incident, while the mapper should map the email of the user to the Reporter Email Address field.
+This playbook assumes that the user requesting the password reset has the same email in Slack / Teams, and in Active Directory / Okta.
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+* Active Directory - Get User Manager Details
```

Comment:
```suggestion
Active Directory - Get User Manager Details
```

Reply:
This was created automatically by the `generate-docs` command of the Demisto SDK. Until there's a solid decision on the new convention, I'm asking to keep this as is